src/ncp/ncp_base.cpp

```diff
@@ -1249,7 +1249,13 @@ otError NcpBase::HandleCommandPropertySet(uint8_t aHeader, spinel_prop_key_t aKe
     otError error = OT_ERROR_NONE;
     PropertyHandler handler = FindSetPropertyHandler(aKey);
 
-    if (handler == NULL)
+    if (handler != NULL)
+    {
+        mDisableStreamWrite = false;
+        error = (this->*handler)();
+        mDisableStreamWrite = true;
```

Comment:
do we want to disable stream write for vendor handlers as well?

Reply:
I changed this in new pushed commit. We are actually enabling it so "set handler" can do logs/prints.